piws/views/logout.py

```diff
@@ -0,0 +1,105 @@
+from cubicweb.web.views.sessions import InMemoryRepositorySessionManager
+from cubicweb.web import LogOut, DirectResponse, Redirect
+from cubicweb.etwist.http import HTTPResponse
+from cubicweb.web.controller import Controller
+from cubes.trustedauth.views import LogoutController
+
+
```

Comment:
The LogOut() command of the PiwsUnloadController will call this class that redirect a user with an expired session to the CW base url?